src/filesystem/README.md

```diff
@@ -9,14 +9,26 @@ Node.js server implementing Model Context Protocol (MCP) for filesystem operatio
 - Move files/directories
 - Search files
 - Get file metadata
+- Load files into model context
 
 **Note**: The server will only allow operations within directories specified via `args`.
 
 ## API
 
 ### Resources
 
-- `file://system`: File system operations interface
```

Comment:
this was/is not true, there are no resources exposed by the filesystem mcp before this PR